purchase_add_product_supplierinfo/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================================
+Add Product Supplierinfo from Purchase Order
+============================================
+
+This module allows in the purchase order, to automatically complete with
+partner, the supplierinfo list for all products which are not related
+to this supplier
+ 
```

Comment:
what do you think of "complete missing entries (product, supplier, quantity) in supplier info list before purchase order confirmation" ?